cmd/tools/rpcwrappers/main.go

```diff
@@ -114,6 +114,7 @@ var (
 		"metricsClient.matching.PollWorkflowTaskQueue": true,
 		"metricsClient.matching.QueryWorkflow":         true,
 		"client.history.GetDLQTasks":                   true,
+		"client.history.DeleteDLQTasks":                true,
```

Comment:
nit: move to L104 to group with other client.history.xxx apis?

Reply:
Done, thanks!